test/metabase/test/util/log.clj

```diff
@@ -166,11 +166,14 @@
   logger is defined for them).
 
     (mt/set-ns-log-level! 'metabase.query-processor.middleware.cache :debug)"
-  [ns-symb new-level :- LogLevelKeyword]
-  (let [logger    (get-or-create-ns-logger! ns-symb)
-        new-level (->Level new-level)]
-    (.setLevel logger new-level)
-    (.updateLoggers (logger-context))))
+  ([new-level :- LogLevelKeyword]
+   (set-ns-log-level! (ns-name *ns*) new-level))
+
+  ([ns-symb new-level :- LogLevelKeyword]
+   (let [logger    (get-or-create-ns-logger! ns-symb)
+         new-level (->Level new-level)]
+     (.setLevel logger new-level)
+     (.updateLoggers (logger-context)))))
```

Comment:
Actually, one more unrelated change. I tweaked this REPL util function to add a 1-arity to change the log level for the current namespace